lib/printer.js

```diff
@@ -646,6 +656,43 @@ function genericPrintNoParens(path, options, print) {
 
         return concat(parts);
 
+    // fallthrough into ObjectMethod
+    case "ClassMethod": // Babel 6
+        if (n.static) {
+            parts.push("static ");
+        }
+
+        // fallthrough into ObjectMethod
```

Comment:
This is the same as an ObjectMethod except you can have static at the beginning - not sure if we want to do it like this or just move the static check into object method anyway